darts/models/forecasting/torch_forecasting_model.py

```diff
@@ -1244,6 +1244,138 @@ def lr_find(
             update_attr=False,
         )
 
+    @random_method
+    def scale_batch_size(
+        self,
+        series: Union[TimeSeries, Sequence[TimeSeries]],
+        val_series: Union[TimeSeries, Sequence[TimeSeries]],
+        past_covariates: Optional[Union[TimeSeries, Sequence[TimeSeries]]] = None,
+        future_covariates: Optional[Union[TimeSeries, Sequence[TimeSeries]]] = None,
+        val_past_covariates: Optional[Union[TimeSeries, Sequence[TimeSeries]]] = None,
+        val_future_covariates: Optional[Union[TimeSeries, Sequence[TimeSeries]]] = None,
+        trainer: Optional[pl.Trainer] = None,
+        verbose: Optional[bool] = None,
+        epochs: int = 0,
+        max_samples_per_ts: Optional[int] = None,
+        num_loader_workers: int = 0,
+        method: Literal["fit", "validate", "test", "predict"] = "fit",
```

Comment:
"test" is not supported for darts. 
"predict" would require a datamodule for prediction

in my opinion, we should only support "fit" and "predict", since we use the same batch size for train and val.